File: src/frds/measures/modified_merton/matlab/ModSingleCohortSolution.m
function [err] = ModSingleCohortSolution(b, param, N, Nsim2, E, sigE)

fs = b(1); 
param(3) = b(2); 

%{
skip = 0; 
sig = b(2);
esig = 0; 
if sig < 0
    sig = 0; 
    esig = 9999999; 
    skip = 1; 
end
eA = 0; 
if fs < 0
    fs = 0; 
    eA = 9999999; 
    skip = 1; 
end
%}

[Lt, Bt, Et, LH, BH, EH, sigEt, mFt, def, mdef, face, FH, Gt, mu, F, sigLt] = ModSingleCohortComputation(fs, param, N, Nsim2);  
err = [E - Et; sigE-sigEt]; 

File: src/frds/measures/modified_merton/mod_single_cohort_solution.py
from frds.measures.modified_merton.mod_single_cohort_computation import (
    mod_single_cohort_computation,
)


def mod_single_cohort_solution(b, param, N, Nsim2, E, sigE):
    fs = b[0]
    param[2] = b[1]

    # fmt: off
    Lt, Bt, Et, LH, BH, EH, sigEt, mFt, default, mdef, face, FH, Gt, mu, F, sigLt = mod_single_cohort_computation(fs, param, N, Nsim2)
    # fmt: on
    err = [E - Et, sigE - sigEt]

    return err
